bas/Model/Recorders.bas

```diff
@@ -447,244 +447,851 @@ source "../Scripts/Recorders.tcl"; # user recorders
 *#
 *# ShellMITC4
 *#
+
 *if(cntShell!=0)
+*#-------------------------------------------------------------------------------------------------------------------------
+*if(GenData(Stresses_in_layer_A,int)==1||GenData(Stresses_in_layer_B,int)==1||GenData(Stresses_in_layer_C,int)==1||GenData(Strains_in_layer_A,int)==1||GenData(Strains_in_layer_B,int)==1||GenData(Strains_in_layer_C,int)==1||GenData(Temp_Elong_and_KtKc_in_layer_A,int)==1||GenData(Temp_Elong_and_KtKc_in_layer_B,int)==1||GenData(Temp_Elong_and_KtKc_in_layer_C,int)==1)
+for {set gaussID 1} {$gaussID<=4} {incr gaussID} {
```

Comment:
This looks like a Tcl loop and will cause an error in this bas file.

Reply:
The code is tested and working perfectly. 'for loop' at line number 454 is not part of the 'bas' execution. 
In fact, the line is dumped to the 'tcl project file' and executed with 'tcl'. 
Also, 'bas for loop' makes the code laborious cause it has to loop the elements by the number of gauss points. 
Where now implementation makes the code execute faster and better by looping elements once.

Reply:
good job.